content/en/docs/about/style-guide.md

```diff
@@ -1,10 +1,22 @@
 
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```

Comment:
Please remove this alert. This is affecting the preview deployment.

Reply:
Done